handlers/middlewares.go

```diff
@@ -67,13 +95,14 @@ func (h *Handler) AuthMiddleware(next http.Handler) http.Handler {
 	return http.HandlerFunc(fn)
 }
 
-func (h *Handler) validateAuth(provider models.Provider, req *http.Request) bool {
-	if err := provider.GetSession(req); err == nil {
+func (h *Handler) validateAuth(provider models.Provider, req *http.Request) (bool, error) {
+	err := provider.GetSession(req)
+	if err == nil {
 		// logrus.Debugf("session: %v", sess)
-		return true
+		return true, err
 	}
 	// logrus.Errorf("session invalid, error: %v", err)
-	return false
+	return false, err
```

Comment:
You can delete the unusable comments like seen here.

Reply:
Yes, will remove.